### PR TITLE
Create run_command_in_console_MOD.js

### PR DESCRIPTION
**Pull request by noahalma**
**Please describe the changes this PR makes and why it should be merged:**

Good mod cool

**Status**

- [x] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by RigidStudios** [comment]
i made this

**Comment by noahalma** [comment]
> i made this

NO YOU DIDN'T

**Comment by DevNoahh** [comment]
I made this

**Comment by RigidStudios** [comment]
yes i made it

**Comment by monksharru** [comment]
i made this

**Comment by noahalma** [comment]
I made this

**Comment by DevNoahh** [comment]
He copied and pasted it from me

**Comment by RigidStudios** [comment]
no it was mine!!! >:(

**Comment by monksharru** [comment]
no it was mine

**Comment by DevNoahh** [comment]
Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis Penis penis

**Comment by monksharru** [comment]
good one noah funny laughed

**Comment by RigidStudios** [comment]
OwO

**Comment by monksharru** [comment]
rip naoh

File: actions/run_command_in_console_MOD.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    Variable name:<br>
+    <input class="round" id="varName" />
+    `
+  },
+  init () {},
+
+  action (cache) {
+    const data = cache.actions[cache.index]
+    const messageToSend = this.evalMessage(data.messageToSend, cache)
+    const response = require("child_process").execSync(messageToSend)
```
Comment (Almeeida) [inline actions/run_command_in_console_MOD.js:33]:
```suggestion
    const response = require("child_process").execSync(messageToSend).toString()
```

File: actions/run_command_in_console_MOD.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <input id="messageToSend" class="round" type="text"><br>
+    Store in:<br>
+    <select class="round" id="storage">
+    ${data.variables[0]}
+    </select><br>
+    Variable name:<br>
+    <input class="round" id="varName" />
+    `
+  },
+  init () {},
```
Comment (Almeeida) [inline actions/run_command_in_console_MOD.js:28]:
```suggestion
  },

  init () {},
```

File: actions/run_command_in_console_MOD.js (hunk 1 above)
Comment (RigidStudios) [reply actions/run_command_in_console_MOD.js]: It doesn't return a buffer so it doesn't matter.

**Comment by noahalma** [comment]
> * Fix the ESLint issues
> * Update the docs

I will update the docs once it is accepted, I want to know where it is on the list of mods first

**Comment by RigidStudios** [comment]
@Almeeida ESLint issues fixed.

File: actions/run_command_in_console_MOD.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      </div>
+      Command to run:
+      <input id="messageToSend" class="round" type="text"><br>
+      Store in:<br>
+      <select class="round" id="storage">
+      ${data.variables[0]}
+      </select><br>
+      Variable name:<br>
+      <input class="round" id="varName" />
+      `
```
Comment (Almeeida) [inline actions/run_command_in_console_MOD.js:26]:
```suggestion
<div>
  <p>
    Run a command in your console.<strong> THIS IS VERY DANGEROUS. SET THIS TO "BOT OWNER ONLY"</strong><br><br>
  </p>
</div>
Command to run:
<input id="messageToSend" class="round" type="text"><br>
Store in:<br>
<select class="round" id="storage">
  ${data.variables[0]}
</select><br>
Variable name:<br>
<input class="round" id="varName" />`
```

**Pull request merged**